site/docs/main/config/rate-limiting.md

```diff
@@ -267,6 +267,98 @@ spec:
       port: 80
 ```
 
+#### Descriptor entries
+
+A descriptor is a list of key-value pairs, i.e. entries, that are generated for a request. The entries can be generated based on different criteria. If any entry in a descriptor cannot generate a key-value pair for a given request, then the entire descriptor is not generated (see the [Envoy documentation][8] for more information). Below are the supported types of descriptor entries.
```

Comment:
maybe worth noting a bit about **all** descriptors that can be attached to a request when it is sent to the RLS will be

Reply:
good call, added a sentence on that.